src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryController.js

```diff
@@ -20,7 +20,7 @@
                 action.setCallback(this, function (response) {
                     const state = response.getState();
                     if (state === 'SUCCESS') {
-                        const response = JSON.parse(response.getReturnValue());
+                        var response = JSON.parse(response.getReturnValue());
```

Comment:
Any opinion on creating a new variable instead of recreating response?

Reply:
Yes, STRONG opinion that this is the right way to do it :)